content/docs/sidebar.json

```
@@ -173,6 +173,11 @@
       {
         "label": "Anonymized Usage Analytics",
         "slug": "analytics"
+      },
+      {
+        "label": "Glossary",
+        "slug": "glossary",
+        "source": "basic-concepts"
       }
```

What would it take to reduce this to just `"glossary"`? Moving the `basic-concepts/` folder?

---

I'm not sure if I'm understanding your question 🤔 Are you referring to deleting `source` or renaming the `source` value to `glossary`? The `source` is used to build the "Edit on Github" button. If you're asking about deleting the source, we don't have a `/content/docs/user-guide/glossary.md` so deleting the source would end up creating a 404 link. I linked the `basic-concepts` folder, since that is where we're currently holding all of our tooltip data. If you're asking about renaming the `source` to `glossary`, I suppose we could do that by renaming the `basic-concepts` folder to `glossary` 😃

---

> What would it take to reduce this sidebar entry to just `"glossary"`? Moving the `basic-concepts/` folder?

I think by "sidebar entry" you meant page URL slug?

The sidebar entry *is* Glossary, and the page URL slug *is* "glossary." The "source" folder is only used for the "Edit on GH" button.

---

I was referring to this:
```suggestion
      "glossary",
```

---

> If you're asking about renaming the source to glossary, I suppose we could do that by renaming the basic-concepts folder to glossary 

Right. Should we do that? But again... #550

---

I don't think we'd be able to do that since then the "Edit on Github" link would default to https://github.com/iterative/dvc.org/blob/master/content/docs/user-guide/glossary.md which doesn't exist.

---

I just noticed this. Hmm so it's a special/hardcoded nav entry then. Not sure I love that but oh well, this is merged!

---

Wait no I see. I think it could default to /content/docs/user-guide/glossary/index.md too? Again, this is merged so np but I was thinking what does it take to decouple it completely from `basic-concepts/`

---

confusion related to https://github.com/iterative/dvc.org/pull/2595#issuecomment-877775897